com.gitlab.davem.ClamTk.appdata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.gitlab.davem.ClamTk</id>
+  <launchable type="desktop-id">com.gitlab.davem.ClamTk.desktop</launchable>
+  <name>ClamTk</name>
+  <developer_name>Dave M</developer_name>
+  <summary>An easy to use, light-weight, on-demand virus scanner for Linux systems</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-1.0+</project_license>
+  <url type="homepage">https://gitlab.com/dave_m/clamtk-gtk3</url>
+  <description>
+    <p>ClamTk is a frontend for ClamAV (Clam Antivirus). It is intended to be an easy to use, light-weight, on-demand scanner for Linux systems. It has been ported to Fedora, Debian, RedHat, openSUSE, ALT Linux, Ubuntu, CentOS, Gentoo, Archlinux, Mandriva, PCLinuxOS, Frugalware, FreeBSD, and others.</p>
+    <p>Although its earliest incarnations date to 2003, ClamTk was first uploaded for distribution in 2004 to a rootshell.be account and finally to Sourceforge.net in 2005. At the end of 2013, it was moved to a Google Code page (then to github), then gitlab and Bitbucket. It's now 2019 and for some reason, it's still going. In fact, February 2019 marks 15 years of activity (of being publically available, that is).</p>
```

Comment:
I am not sure if this information is somehow useful for the user. Maybe some details about the ClamAV antivirus itself could be more useful. :-)